fast64_internal/mk64/__init__.py

```diff
@@ -0,0 +1,162 @@
+import bpy
+from bpy.types import PropertyGroup, Operator
+from bpy.utils import register_class, unregister_class
+from .f3d.properties import MK64CourseDLImportSettings, f3d_props_register, f3d_props_unregister
+from .f3d_course_parser import MK64F3DContext, parseCourseVtx
+from ..f3d.f3d_material import createF3DMat
+from ..f3d.f3d_gbi import get_F3D_GBI
+from ..f3d.f3d_parser import getImportData, importMeshC
+from ..panels import MK64_Panel
+from ..utility import prop_split
+from ..utility import raisePluginError
+
+
+class MK64_Properties(PropertyGroup):
+    """Global MK64 Scene Properties found under scene.fast64.mk64"""
+
+    # Import Course DL
+    CourseDLImportSettings: bpy.props.PointerProperty(type=MK64CourseDLImportSettings)
+
+    @staticmethod
+    def upgrade_changed_props():
+        pass
+
+
+class MK64_ImportCourseDL(Operator):
```

Comment:
shouldn´t the operator and panel be in f3d?

Reply:
hmm what do you mean ?

Reply:
the f3d folder I mean